sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1132,6 +1132,12 @@ private void SetNameFieldsIfNull()
 
             ResponseWithHeaders<Stream, BlobDownloadHeaders> response;
 
+            // All BlobRequestConditions are valid.
```

Comment:
if this is the case, is this just here just in case of the future that there are conditions added that are invalid?

Reply:
Yep!